views/support/show.ejs

```diff
@@ -1,93 +1,99 @@
-        <li><a href="<%=complaint.author.id.email%>"><%=complaint.author.id.email%></a></li>
-    </ul>
-</div>
-</div>
-</div>
-<div class="col-sm-12 col-md-8">
-    <div class="col-12" id="user-info">
-    <h2 class="user-details" style="text-align: left;">Ticket Information:</h2>
-    <hr class="style-one">
-    <div class="details">
-        <div style="float: left;">
-            <ul>
-                <li><strong class="ticket-details">Title/Subject:</strong></li>
-                <li><strong class="ticket-details">Created At:</strong></li>
-                <li><strong class="ticket-details">Opened At:</strong></li>
-                <li><strong class="ticket-details">Updated At:</strong></li>
-                <li><strong class="ticket-details">Ticket ID/Ref. No:</strong></li>
-                <li><strong class="ticket-details">Status:</strong></li>
-            </ul>
-        </div>
-        <div style="float: right;">
-            <ul>
-                <li><%=complaint.name%></li>
-                <li><%=complaint.createdAt%></li>
-                <li><%=complaint.openedAt%></li>
-                <li><%=complaint.updatedAt%></li>
-                <li><%=complaint.id%></li>
-                <% if(!complaint.status){ %>
-                   <li>Not Assigned Yet.</li>
-                <% }else{ %>
-                    <li><%=complaint.status%></li>
-                <% } %>
-            </ul>
+    <div class="col-sm-12 col-md-4">
+        <div class="col-12" id="user-info">
+            <h2 class="user-details">User Details:</h2>
+            <img
+                class="img-fluid"
+                id="user-image"
+                src="https://images.unsplash.com/photo-1504639725590-34d0984388bd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
+                alt="user-profile-picture"
+                id="profile-picture"
+            />
+            <div class="details">
+                <ul>
+                    <li><strong>Username: </strong><a href="">@<%=complaint.author.username%></a></li>
+                    <li><strong>Name: </strong><%=complaint.author.firstName%> <%=complaint.author.lastName%></li>
+                    <li><strong>Email: </strong></li>
+                    <li><a href="<%=complaint.author.id.email%>"><%=complaint.author.email%></a></li>
+                </ul>
+            </div>
         </div>
     </div>
-    <hr>
-    <div class="row">
-        <div class="col-sm-12 col-md-6">
-            <form action="/support/<%=complaint._id%>?_method=PUT" method="POST">
-                <div class="form-group">
-                    <label for="" class="ticket-details">Select Status:</label>
-                    <select class="form-control" name="status" id="statusSelect">
-                      <option value="">--Select Status--</option>  
-                      <option value="Open">Open</option>
-                      <option value="Waiting List">Waiting List</option>
```

Comment:
values should never contain empty spaces use dashes - instead like so:  waiting-list